src/Dropdown/__tests__/Dropdown.test.tsx

```diff
@@ -43,19 +53,51 @@ describe('Dropdown', function () {
   });
   it('contains the expected elements', function () {
     const dropdownOptionsCount = getAllByRole('option').length;
-    strictEqual(dropdownOptionsCount, 3);
+    strictEqual(dropdownOptionsCount, 4);
```

Comment:
You might want to compare this with `options.length`, rather than a magic number :)

Reply:
Ah! Good point. I updated this through commit c29d078.